ContainerFile

```diff
@@ -0,0 +1,18 @@
+# File for creating a container of this testsuite.
+FROM registry.access.redhat.com/ubi8/openjdk-8:latest
+USER root
+RUN whoami
+
+# Install required utilities
+RUN microdnf -y install util-linux
+
+# Create the directory structure for the testing.
+RUN mkdir -p /tmp/rhscl_openshift_dir/openjdk
+WORKDIR /tmp/rhscl_openshift_dir/openjdk
+COPY . .
+
+# Update the permissions on the testsuite
+RUN chgrp -R 0 /tmp/rhscl_openshift_dir/openjdk && \
+    chmod -R g=u /tmp/rhscl_openshift_dir/openjdk && \
+    chgrp -R 0 /home && \
```

Comment:
Wou. Why is this file suddenly needed? It is for cpu?

Reply:
I mena the whole [ContainerFile](https://github.com/rh-openjdk/jdkContainerOcpTests/pull/5/files/e464ad9cabd3fd0d5448d7002a0f19ad3bcb1d65#diff-1f41dac8c7830a4c112e8f6ba376c69c7687fdb7e1c66919a1c12b8749303264) not that tht quoted part

Reply:
The ContainerFile is needed for building the container used for the InterOp testing in the OpenShift CI.
It creates this base image used in their testing: https://quay.io/repository/rhopenjdkqa/rh_jdk_ocp_testsuite

Reply:
You know better. ty!